src/routes/radio/+page.svelte

```diff
@@ -1,8 +1,104 @@
-<script>
-  import CommonHead from '$lib/components/CommonHead/CommonHead.svelte';
-  import NotFinished from '$lib/components/NotFinished/NotFinished.svelte';
+<script lang="ts">
   import { m } from '$lib/paraglide/messages';
+  import CommonHead from '$lib/components/CommonHead/CommonHead.svelte';
+  import tireTrack from '$lib/assets/images/original/tire_tracks.jpg';
+  import Player from '$lib/components/Radio/Player.svelte';
+  import { writable } from 'svelte/store';
+
+  const nowPlaying = writable<string | null>('Turbo Groove');
 </script>
 
 <CommonHead>{m['radio.title']()}</CommonHead>
-<NotFinished />
+
+<div class="relative min-h-screen overflow-hidden bg-gray-100 py-8 dark:bg-gray-900">
+  <img
+    src={tireTrack}
+    alt="Tire background"
+    class="pointer-events-none absolute inset-0 h-full w-full object-cover opacity-100 dark:opacity-70"
+  />
+  <div class="pointer-events-none absolute inset-0 bg-black/90 dark:bg-black/70"></div>
+
+  <div class="relative z-10 mx-auto w-full px-3">
+    <Player streamUrl="/stream" stationName="ASEAN Motor Club Radio">
+      <span slot="current">{$nowPlaying}</span>
+    </Player>
+  </div>
+
+  <div
```

Comment:
You can use `<Card />` for this. also it broken on light mode

![Image](https://github.com/user-attachments/assets/6fbcd470-af9f-4ce7-b444-696f8adfdd2a)